docs/advanced/install_packages.md

```diff
@@ -15,11 +15,10 @@ A typical case is the installation of Python packages, which requires to run pip
 
 If this fails because you are trying to install an updated version of a package that already exists in CVMFS, you will need to add the `--upgrade` flag.
 
-Then, it would be necessary to add the local installation path to `PYTHONPATH`, by creating a bash startup script that configures that variable (don't forget to call this startup script in the session configuration menu):
+After the desired Python packages have been installed on CERNBox, they need to be added to the PYTHONPATH when starting a SWAN session. This can be easily 
+achieved by selecting the option "Use Python packages installed on CERNBox" under the "Software stack" section of the SWAN web form.
 
-``` python
-export PYTHONPATH=$CERNBOX_HOME/.local/lib/python3.5/site-packages:$PYTHONPATH
-```
+![][user_python_packages]
```

Comment:
Can we just show the "Software stack" section of the form in this image? Otherwise it will easily get outdated.